pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -208,12 +215,20 @@ def batch_sampler(
 
     if num_batches is None:
         if impl_params:
+            # https://github.com/pmeier/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/train.lua#L48
+            # The num_iterations are split up into multiple epochs with corresponding num_batches:
+            # 50000 = 25 * 2000
+            # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/stylization_train.lua#L30
+            # The num_iterations are split up into multiple epochs with corresponding num_batches:
+            # 3000 = 10 * 300
```

Comment:
You should hint here where the number of epochs is defined.

```suggestion
            # The num_iterations are split up into multiple epochs with corresponding num_batches:
            # 50000 = 25 * 2000
            # https://github.com/pmeier/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/train.lua#L48
            # 3000 = 10 * 300
            # https://github.com/pmeier/texture_nets/blob/b2097eccaec699039038970b191780f97c238816/stylization_train.lua#L30
```